src/pyload/core/api/__init__.py

```diff
@@ -1215,15 +1236,15 @@ def get_account_types(self):
 
     @legacy("updateAccount")
     @permission(Perms.ACCOUNTS)
-    def update_account(self, plugin, account, password=None, options={}):
+    def update_account(self, plugin: str, account: str, password: str = None, options: dict[str: Any] = {}) -> None:
```

Comment:
I understand the code analysis is throwing a warning here, but I think for now we should keep the default arguments as they were before.